frontend/src/router/router.jsx

```diff
@@ -27,9 +27,8 @@ const RouterComponent = () => { // eslint-disable-line arrow-body-style
     <Router history={history}>
       <Switch>
         <Route exact path="/" component={Splash} />
```

Comment:
Yarn throws linter error of unused variable on line 12: `AllData`.

Reply:
Linter issue (unused import) is now resolved. All tests pass now.